src/Tribe/View_Helpers.php

```diff
@@ -295,8 +295,8 @@ public static function constructCountries( $postId = '', $useDefault = true ) {
 				if ( $defaultCountry && $defaultCountry[0] != '' ) {
 					$selectCountry = array_shift( $countries );
 					asort( $countries );
-					$countries = array( $defaultCountry[0] => __( $defaultCountry[1], 'tribe-common' ) ) + $countries;
-					$countries = array( '' => __( $selectCountry, 'tribe-common' ) ) + $countries;
+					$countries = array( $defaultCountry[0] => $defaultCountry[1] ) + $countries;
```

Comment:
Hmm, there's differences between the two (`array_merge()` vs an array union using `+`) though I guess in this case both would have the same result ... mostly though I just wanted to resolve the i18n issue and not change any other logic.

I'm happy to make the change but isn't `+` preferred? It's far more concise.